x-pack/plugins/rollup/public/crud_app/sections/job_create/job_create.js

```diff
@@ -159,7 +159,7 @@ export class JobCreateUi extends Component {
   requestIndexPatternValidation = debounce((resetDefaults = true) => {
     const indexPattern = this.getIndexPattern();
 
-    const lastIndexPatternValidationTime = (this.lastIndexPatternValidationTime = Date.now());
+    const lastIndexPatternValidationIdx = ++this.lastIndexPatternValidationIdx;
```

Comment:
Not relying on the `Date` for this as we can simply use an incremental counter. Better for the tests too.